practice-two/src/index.html

```diff
@@ -12,11 +12,11 @@
   <!-- sign up start  -->
   <div class="sign-up">
     <div class="container sign-up">
-    <p class="sign-up__text">Sign up and get 20% off to your first order. <a class="sign-up__link"
-        href="javascript:void(0)">Sign Up
-        Now</a></p>
-    <a href="javascript:void(0)"><img class="sign-up__icon" src="assets/images/headers/off.svg" alt="off"></a>
-  </div>
+      <p class="sign-up__text">Sign up and get 20% off to your first order. <a class="sign-up__link"
```

Comment:
Please check class name. It should be sign-up__content, sign-up__description or sign-up__advertise